### ISSUE TITLE
Uncaught TypeError: Cannot read property 'buffer' of undefined

### ISSUE THREAD

**Issue by darwinrc** [philnash/ruby-quick-test#12]
[Enter steps to reproduce below:]

1. ...
2. ...

**Atom Version**: 0.187.0
**System**: linux 3.16.0-33-generic
**Thrown From**: [ruby-quick-test](https://github.com/philnash/ruby-quick-test) package, v0.3.0

### Stack Trace

Uncaught TypeError: Cannot read property 'buffer' of undefined

```
At /home/darwin/.atom/packages/ruby-quick-test/lib/ruby-quick-test-view.coffee:62

TypeError: Cannot read property 'buffer' of undefined
  at RubyQuickTestView.module.exports.RubyQuickTestView.activeFile (/home/darwin/.atom/packages/ruby-quick-test/lib/ruby-quick-test-view.coffee:62:61)
  at RubyQuickTestView.module.exports.RubyQuickTestView.testFileType (/home/darwin/.atom/packages/ruby-quick-test/lib/ruby-quick-test-view.coffee:65:19)
  at RubyQuickTestView.module.exports.RubyQuickTestView.runTests (/home/darwin/.atom/packages/ruby-quick-test/lib/ruby-quick-test-view.coffee:48:13)
  at atom-workspace.<anonymous> (/home/darwin/.atom/packages/ruby-quick-test/lib/ruby-quick-test-view.coffee:1:1)
  at atom-workspace.handler (/usr/share/atom/resources/app/src/space-pen-extensions.js:115:32)
  at atom-workspace.jQuery.event.dispatch (/usr/share/atom/resources/app/node_modules/space-pen/vendor/jquery.js:4681:9)
  at atom-workspace.elemData.handle (/usr/share/atom/resources/app/node_modules/space-pen/vendor/jquery.js:4359:46)
  at CommandRegistry.module.exports.CommandRegistry.handleCommandEvent (/usr/share/atom/resources/app/src/command-registry.js:246:29)
  at /usr/share/atom/resources/app/src/command-registry.js:3:61
  at CommandPaletteView.module.exports.CommandPaletteView.confirmed (/usr/share/atom/resources/app/node_modules/command-palette/lib/command-palette-view.js:120:32)
  at CommandPaletteView.module.exports.SelectListView.confirmSelection (/usr/share/atom/resources/app/node_modules/atom-space-pen-views/lib/select-list-view.js:338:21)
  at space-pen-div.atom.commands.add.core:confirm (/usr/share/atom/resources/app/node_modules/atom-space-pen-views/lib/select-list-view.js:109:19)
  at CommandRegistry.module.exports.CommandRegistry.handleCommandEvent (/usr/share/atom/resources/app/src/command-registry.js:246:29)
  at /usr/share/atom/resources/app/src/command-registry.js:3:61
  at KeymapManager.module.exports.KeymapManager.dispatchCommandEvent (/usr/share/atom/resources/app/node_modules/atom-keymap/lib/keymap-manager.js:558:16)
  at KeymapManager.module.exports.KeymapManager.handleKeyboardEvent (/usr/share/atom/resources/app/node_modules/atom-keymap/lib/keymap-manager.js:396:22)
  at HTMLDocument.module.exports.WindowEventHandler.onKeydown (/usr/share/atom/resources/app/src/window-event-handler.js:182:20)

```

### Commands

```
     -2:30.8 emmet:select-previous-item (ol.tree-view.full-menu.list-tree.has-collapsable-children.focusable-panel)
     -2:28.0 command-palette:toggle (ol.tree-view.full-menu.list-tree.has-collapsable-children.focusable-panel)
     -2:22.9 core:confirm (atom-text-editor.editor.mini)
     -2:22.9 settings-view:open (ol.tree-view.full-menu.list-tree.has-collapsable-children.focusable-panel)
     -1:57.2 core:confirm (atom-text-editor.editor.mini)
     -1:25.8 ruby-test:test-all (atom-workspace.workspace.scrollbars-visible-always.theme-seti-syntax.theme-seti-ui)
     -0:43.9 ruby-test:test-file (atom-workspace.workspace.scrollbars-visible-always.theme-seti-syntax.theme-seti-ui)
     -0:19.6 atom-terminal:open (atom-workspace.workspace.scrollbars-visible-always.theme-seti-syntax.theme-seti-ui)
     -0:14.5 command-palette:toggle (div.settings-view.pane-item)
  3x -0:10.9 core:move-down (atom-text-editor.editor.mini)
  4x -0:09.6 core:move-up (atom-text-editor.editor.mini)
  2x -0:06.3 core:move-down (atom-text-editor.editor.mini)
  5x -0:04.8 core:backspace (atom-text-editor.editor.mini)
     -0:02.3 core:confirm (atom-text-editor.editor.mini)
     -0:02.3 ruby-quick-test:run-tests (div.settings-view.pane-item)
```

### Config

```json
{
  "core": {
    "projectHome": "/home/darwin/dev",
    "disabledPackages": [
      "git-review",
      "ruby-test"
    ],
    "themes": [
      "seti-ui",
      "seti-syntax"
    ]
  }
}
```

### Installed Packages

```coffee
# User
atom-terminal, v0.7.0
atom-typescript, v1.5.1
autocomplete-plus, v2.6.0
autocomplete-ruby, v0.0.1
color-picker, v1.5.0
emmet, v2.3.4
git-log, v0.3.0
javascript-snippets, v1.0.0
language-coffee-script, v0.39.0
language-javascript, v0.63.0
linter, v0.12.0
linter-ruby, v0.1.4
ruby-block, v0.3.3
ruby-block-converter, v3.3.0
ruby-define-method, v0.2.0
ruby-quick-test, v0.3.0
ruby-strftime-reference, v0.2.0
seti-syntax, v0.3.3
seti-ui, v0.6.3

# Dev
No dev packages
```

**Comment by philnash** [comment]
Hummm... Wonder what's going on there?

Do you have an open test file when you run this? Is there anything else that may lead to this at all? Is it happening consistently? Was it working with older versions of Atom?

(sorry about all the questions, haven't looked at this project for a while)

**Comment by philnash** [comment]
This has now been fixed as of Ruby Quick Test 0.4.0, please upgrade. This was due to deprecated methods being removed.

**Comment by darwinrc** [comment]
Thanks @philnash 

I'll upgrade right now!
